torchgeo/datasets/esri2020.py

```diff
@@ -136,3 +137,52 @@ def _download(self) -> None:
     def _extract(self) -> None:
         """Extract the dataset."""
         extract_archive(os.path.join(self.root, self.zipfile))
+
+    def plot(  # type: ignore[override]
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample returned by :meth:`RasterDataset.__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional string to use as a suptitle
+
+        Returns:
+            a matplotlib Figure with the rendered sample
+
+        .. versionchanged:: 0.3
+            Method now takes a sample dict, not a Tensor. Additionally, it is possible
+            to show subplot titles and/or use a custom suptitle.
```

Comment:
This can be removed for this specific dataset since it hasn't yet been released. I.e. the entire class is new in 0.3, so it can't also be changed in 0.3.